service/hook/parser/parse.go

```diff
@@ -232,9 +232,17 @@ func (p *parser) Parse(req *http.Request, secretFunc func(string) string) (*core
 		}
 		return hook, repo, nil
 	case *scm.PullRequestHook:
-		if v.Action != scm.ActionOpen && v.Action != scm.ActionSync {
+		switch v.Action {
+		case scm.ActionOpen:
```

Comment:
you can shorten with `case scm.ActionOpen, scm.ActionClose, scm.ActionSync:`

Reply:
Done; seems to compile. Will test it .. shortlyish.